app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @current_user = current_user
+    @post = Post.find(params[:post_id])
```

Comment:
- [ Optional ] I would like to suggest that you use strong parameters in your controllers when expecting input from the `forms` this will prevent assigning request parameters to objects unless they have been explicitly permitted by your controller. Please see this article for more details on this [Click Here >>](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

## Example code
```ruby
  class CommentsController < ApplicationController

    def create
      @comment = @current_user.comments.new(params)
      @comment.post_id = params[:post_id]

      if @comment.save
        redirect_to post_path(@comment.post)
      else
        @article = Post.find(params[:post_id])
        redirect_to post_path(@post)
      end
    end


  private 
  def comment_params
    params.require(:comment).permit(:text)
  end

  end
```